emulators/qemu-guest-agent/src/opnsense/mvc/app/models/OPNsense/QemuGuestAgent/QemuGuestAgent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ *    Copyright (C) 2021 Deciso B.V.
+ *    Copyright (C) 2015 Deciso B.V.
```

Comment:
Should this have been your copyright line?

Reply:
Not this one, but the one above.

Reply:
Fixed, thanks.